driven/storage/adapter.go

```diff
@@ -462,3 +461,79 @@ func (sa *Adapter) DeleteSurvey(user *model.User, id string) error {
 
 	return nil
 }
+
+// Get SurveyResponse
+func (sa *Adapter) GetSurveyResponse(user *model.User, id string) (*model.SurveyResponse, error) {
+
+	filter := bson.M{"_id": id}
```

Comment:
This is something that we will want to be quite careful about. At the moment, this API could be used by any user to get the response of any other user if they know the `id` of the response. This would be a privacy/security concern, so we will need to ensure that the user ID (subject) is included in the filter like we do for `UpdateSurveyResponse` below.